facia/public/facebook_ia_remote.html

```diff
@@ -0,0 +1,172 @@
+                krux.setAttribute("class", "kxct");
+                krux.setAttribute("data-id", "J_RxuxWJ");
+                krux.setAttribute("data-timing", "async");
+                krux.setAttribute("data-version", "1.9");
+                krux.setAttribute("type", "text/javascript");
+
+                krux.innerHTML = "window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]); " +
+                    "(function(){ " +
+                    "var k=document.createElement('script');k.type='text/javascript';k.async=true; " +
+                    "var m,src=(m=location.href.match(/\bkxsrc=([^&]+)/))&&decodeURIComponent(m[1]); " +
+                    "k.src = /^https?:\\/\\/([a-z0-9_\\-\\.]+\\.)?krxd\\.net(:\\d{1,5})?\\//i.test(src) ? src : src === 'disable' ? '' : " +
+                    "(location.protocol==='https:'?'https:':'http:')+'//cdn.krxd.net/controltag?confid=J_RxuxWJ' ; " +
+                    "var s=document.getElementsByTagName('script')[0];s.parentNode.insertBefore(k,s); }());";
+
+                document.body.appendChild(krux);
+            }
+
+            function buildTargetSize(size) {
+                var formatted = size.replace(/\s*,\s*/g, "],[").replace(/\s*x\s*/g, ",");
+                return "[[" + formatted + "]]";
+            }
+
+            function buildCustomTargeting() {
+                var attr, i, key, len, match, value;
+                var targeting = "";
+                var ref = slot.attributes;
+                for (i = 0, len = ref.length; i < len; i++) {
+                    attr = ref[i];
+                    match = attr.name.match(/^data-ad-target-(.+)$/);
+                    if (match != null) {
+                        key = match[1];
+                        value = attr.value.replace(/\s*,\s*/g, "','");
+                        targeting += ".setTargeting('" + key + "',['" + value + "'])";
+                    }
+                }
+                return targeting;
+            }
+
+            function buildSegments() {
+                var filling, kruxSegments, output, ref;
+                var targeting = "";
+                kruxSegments = (ref = localStorage.getItem('kxsegs')) != null ? ref : "";
+                if (kruxSegments !== "") {
+                    filling = kruxSegments.split(',').join("','");
+                    output = "['" + filling + "']";
+                    targeting += ".setTargeting('x', " + output + ")";
+                }
+                return targeting;
+            }
+
+            var pageLevelCodePart1 = "var googletag = googletag || {}; googletag.cmd = googletag.cmd || []; (function() { var gads = document.createElement(\"script\"); gads.async = true; gads.type = \"text/javascript\"; var useSSL = \"https:\" == document.location.protocol; gads.src = (useSSL ? \"https:\" : \"http:\") + \"//www.googletagservices.com/tag/js/gpt.js\"; var node = document.getElementsByTagName(\"script\")[0]; node.parentNode.insertBefore(gads, node); })();";
```

Comment:
the variables `pageLevelCodePart1` and `pageLevelCodePart2` are only used once as well,  I would move them to be inside the function that they are used. This means they only stay within that function scope, and also helps with readability.